app/templates/requirement/form.plush.html

```diff
@@ -1,10 +1,31 @@
-
+<div class="d-flex justify-content-between mb-4">
+    <div class="w-48">
         <%= f.InputTag("Title") %>
-        <%= f.TextArea("Description", {rows: 5}) %>
+    </div>
+    <div class="w-48" >
+        <%= f.SelectTag("ServiceDepartmentID", {options: departments , label: "Service Area",class:"form-select" }) %>
+    </div>
```

Comment:
Take a look at the Bootstrap doc (Grid System). We can get to accommodate those divs without extra classes.